odl_video/settings.py

```diff
@@ -438,15 +438,8 @@
     'VIDEO_S3_SUBTITLE_BUCKET',
     'VIDEO_S3_WATCH_BUCKET',
     'ENABLE_VIDEO_PERMISSIONS',
-]
-
-if ENABLE_VIDEO_PERMISSIONS:
-    MANDATORY_SETTINGS += [
-        'YT_ACCESS_TOKEN',
-        'YT_REFRESH_TOKEN',
-        'YT_CLIENT_ID',
-        'YT_CLIENT_SECRET',
-        'YT_PROJECT_ID',
+    'YT_ACCESS_TOKEN',
+    'YT_REFRESH_TOKEN',
```

Comment:
Just wondering: why are these two settings required to run the app, and why are the other 3 YT not required?

Reply:
The optional 3 settings are used only when running the `oauthtokens` command, which is what will give you values for `YT_ACCESS_TOKEN` and `YT_REFRESH_TOKEN`.  These two are required for handling all Youtube API requests.

Reply:
Actually `YT_CLIENT_ID` and `YT_CLIENT_SECRET` are required too, not sure how I missed that.  Only `YT_PROJECT_ID` is optional.  Will update.

Reply:
Updated `app.json` and `settings.py` to indicate that `YT_CLIENT_ID` and `YT_CLIENT_SECRET` are required.